packages/modular-scripts/src/__tests__/select.test.ts

```diff
@@ -0,0 +1,398 @@
+import execa from 'execa';
+import path from 'path';
+import fs from 'fs-extra';
+
+import getModularRoot from '../utils/getModularRoot';
+import { createModularTestContext, runModularPipeLogs } from '../test/utils';
+
+// Temporary test context paths set by createTempModularRepoWithTemplate()
+let tempModularRepo: string;
+
+describe('select', () => {
+  const fixturesFolder = path.join(
+    getModularRoot(),
+    '__fixtures__',
+    'ghost-building',
+  );
+
+  beforeAll(() => {
+    tempModularRepo = createModularTestContext();
+    fs.copySync(fixturesFolder, tempModularRepo);
+
+    // Create git repo & commit
+    if (process.env.GIT_AUTHOR_NAME && process.env.GIT_AUTHOR_EMAIL) {
+      execa.sync('git', [
+        'config',
+        '--global',
+        'user.email',
+        `"${process.env.GIT_AUTHOR_EMAIL}"`,
+      ]);
+      execa.sync('git', [
+        'config',
+        '--global',
+        'user.name',
+        `"${process.env.GIT_AUTHOR_NAME}"`,
+      ]);
+    }
+    execa.sync('git', ['init'], {
+      cwd: tempModularRepo,
+    });
+
+    execa.sync('yarn', {
+      cwd: tempModularRepo,
+    });
+
+    execa.sync('git', ['add', '.'], {
+      cwd: tempModularRepo,
+    });
+
+    execa.sync('git', ['commit', '-am', '"First commit"'], {
+      cwd: tempModularRepo,
+    });
+  });
+
+  it('selects nothing when everything committed', () => {
+    const result = runModularPipeLogs(tempModularRepo, 'select --changed');
```

Comment:
Is it possible to run the select command and mock getModularRepo as with the esmView tests instead of using runModularPipeLogs? I feel like we should apply the new approach to new tests instead of expanding the use of execa 
We haven't replaced tests that pipe logs to check stdout/err yet so it will require some thinking about

Reply:
I'll capture this in a separate ticket - trying to keep the PRs single-themed.

Reply:
Done.